addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1270,11 +1270,28 @@ var UrlWidget = InputField.extend({
     /**
      * Urls are links in readonly mode.
      *
+     * You can use attributes to customize its visualization:
```

Comment:
When possible we avoid using personal pronoun in the code.

Talk about the method or the class, not the user.